test/pretty-print.test.js

```diff
@@ -93,3 +93,31 @@ test('pretty print - wildcard routes', t => {
     t.equal(tree, expected)
   })
 })
+
+test('pretty print - empty plugins', t => {
+  t.plan(2)
+
+  const fastify = Fastify()
+  fastify.ready(() => {
+    const tree = fastify.printPlugins()
+    t.is(typeof tree, 'string')
+    t.match(tree, 'bound root')
+  })
+})
+
+test('pretty print - nested plugins', t => {
+  t.plan(4)
+
+  const fastify = Fastify()
+  fastify.register(async function foo (instance) {
+    instance.register(async function bar () {})
+    instance.register(async function baz () {})
+  })
+  fastify.ready(() => {
+    const tree = fastify.printPlugins()
+    t.is(typeof tree, 'string')
+    t.match(tree, 'foo')
+    t.match(tree, 'bar')
+    t.match(tree, 'baz')
+  })
```

Comment:
can you please include the full output?

Reply:
I didn't because the output is non deterministic -- it includes timing information about how long it took to load each of the plugins. I figured mocking that out would be quite brittle, and coupling directly to the internal implementation in avvio is kind of undesirable anyways, so I figured these coarser assertions were a happy middle ground.

Reply:
ok, great!